.travis.yml

```diff
@@ -24,6 +24,7 @@ addons:
 
 install:
   - go get -u github.com/bazelbuild/buildifier/buildifier
+  - curl -LO https://storage.googleapis.com/container-diff/latest/container-diff-linux-amd64 && chmod +x container-diff-linux-amd64 && sudo mv container-diff-linux-amd64 /usr/local/bin/container-diff
```

Comment:
Just a note we all need this on our machines something to mention in https://github.com/GoogleCloudPlatform/base-images-docker/blob/master/CONTRIB.md

Reply:
Done.